nuc_morph_analysis/analyses/volume/add_growth_features.py

```diff
@@ -185,6 +308,8 @@ def plot_fit_parameter_distribution(
         Flag for whether to separate data out by colony
```

Comment:
Missing a description of `feature` as a parameter for this function. 
I notice down below that feature is not used when `allmodels_flag==True`.